WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -1616,6 +1622,15 @@ function(account, id) {
 	return identity && (typeof prop == "string" ? identity[prop] : identity[prop]());
 };
 
+ZmAccountsPage.prototype.__mandatoryEmail =
+function(str) {
+	if (!str || str.trim().length == 0) {
+		this._errorMsg = AjxStringUtil.htmlEncode(AjxMessageFormat.format(ZmMsg.fieldNameIsARequiredField, [AjxMessageFormat.format(ZmMsg.emailAddr)]));
```

Comment:
we are taking values from translation strings only and not using any user data so do we need htmlEncode?

Reply:
Fixed it. Thank you for the catch.